.github/scripts/run_ui_tests.sh

```diff
@@ -0,0 +1,30 @@
+sudo apt update
+apt install net-tools
+apt-get install wget
+mkdir tests
+
+#download connector ui tests
+mv .github/testcases/ui_tests.json $HOME/target/test/connector_tests.json
+
+#install geckodriver
+wget -c https://github.com/mozilla/geckodriver/releases/download/v0.33.0/geckodriver-v0.33.0-linux-aarch64.tar.gz && tar -xvzf geckodriver*
```

Comment:
The github runner images already have firefox/gecko drivers. Can you check if they are sufficient?

https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#browsers-and-drivers

```
Selenium server 4.10.0
Mozilla Firefox 114.0.2
Geckodriver 0.33.0
```